packaging/windows/buildpackage.sh

```diff
@@ -85,5 +85,12 @@ if [ $? -eq 0 ]; then
 fi
 popd > /dev/null
 
+echo "Packaging zip archive"
+pushd "${BUILTDIR}" > /dev/null
+cp $(find "${SRC_DIR}/thirdparty/download/windows/" -name '*.dll') .
```

Comment:
Why not `find ... -exec`?

Reply:
You mean `find ... -exec cp {} . +`? That should work too.

Reply:
To use the `+` variant, I would have to use `cp -t` which is only available on linux afaik. The `;` variant is probably slower than command substitution but it should be safer for the case where `find` finds a lot of files.